cli/Makefile

```diff
@@ -1,21 +1,25 @@
 SHELL := /bin/bash
 
 # Changing this value will trigger a new release
-VERSION=v0.6.2
+VERSION=v0.7.0
 BINARY=bin/cft
 GITHUB_REPO=github.com/GoogleCloudPlatform/cloud-foundation-toolkit
 PLATFORMS := linux windows darwin
 BUILD_DIR=./bin
+SCHEMA_DIR=./bpmetadata/schema
```

Comment:
Do we use this?

Reply:
We should. I updated the script to use it.